ipmiMgrApp/Db/mch_nat.substitutions

```diff
@@ -26,24 +26,24 @@
 
 file "mch_base.db"
 {
-    pattern { suffix       , fruid }
-            { $(id)Base    , 3     }
+    pattern { dev       , fruid }
+            { $(dev)Base, 3     }
 }
 
 file "mch_clk_switch.db"
 {
-    pattern { suffix       , fruid }
-            { $(id)ClkSw   , 60    }
+    pattern { dev        , fruid }
+            { $(dev)ClkSw, 60    }
 }
 
 file "mch_pci_hub.db"
 {
-    pattern { suffix       , fruid }
-            { $(id)PciHub  , 61    }
+    pattern { dev          , fruid }
+            { $(dev)PCI_Hub, 61    }
```

Comment:
`PCI_Hub` is not consistent with the documentation (which lists it as `PciHub`).

According to the current substitution, this will be also prefixed by `MCH`, because we use `$(dev)` here, as far as I understand. This is also not what is documented in `PVList.md`. I'm not sure which one you intended to use.

Reply:
yeah, indeed. 
You're right about the prefix. It should be `MCH`. 
Now, I'm thinking if it isn't better to add an underscore after the MCH prefix in it's substitution tree e.g. `MCH_PCI_Hub`